src/Symfony/Component/VarExporter/LazyGhostTrait.php

```diff
@@ -148,7 +148,8 @@ public function &__get($name): mixed
             }
             $accessor = Registry::$classAccessors[$scope] ??= Registry::getClassAccessors($scope);
 
-            return $accessor['get']($this, $name, null !== $readonlyScope);
+            $var = $accessor['get']($this, $name, null !== $readonlyScope);
+            return $var;
```

Comment:
Can we leave a clarifying comment for the next person who attempts to inline that variable?